### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by mahammad-mostafa**
- Built project with `create-react-app` & cleared default files
- Used `React Router` for controlling browser navigation
- Built `Layout` component for all route paths in app
- Created `Header` component with navigation links
- Implemented `Books` list operations logic (adding / deleting)
- Created `Book` component item with deletion handling event
- Created basic `Categories` component for categories page

### CODE REVIEW

File: src/components/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      const book = { id: v4(), title: form.title.value, author: form.author.value };
+      setItems([...items, book]);
+      form.reset();
+    }
+  };
+
+  return (
+    <section>
+      <h2>Books List</h2>
+      <ul>{items.map((item) => <Book key={item.id} book={item} handler={handleDelete} />)}</ul>
```
Comment (1GORDON) [inline src/components/books.js:21]: Great job @mahammad-mostafa 🤝

- [ ] As stated in the project requirements, kindly ensure the books component renders both the booklist and the book components properly. The book component should contain the delete button which is not rendered in the UI.
book-list
- Kindly use the project requirements below
book

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mahammad-mostafa  ✋

Good job so far!

## Highlights!
- Used react-router ✔️ 
- Project initialized with create-react-app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books.js (hunk 1 above)
Comment (mahammad-mostafa) [reply src/components/books.js:21]: Thanks @1GORDON for feedback, But I understood that the button should be added with the `Book` component item. That is why it appears whenever you add a book to the list.

Comment (1GORDON) [reply src/components/books.js:21]: Thank you for your response @mahammad-mostafa. However, at the moment the buttons are not functional. We only need the UI to be rendered properly. At the moment we won't be adding and deleting books.

**Review by jerryowusu — APPROVED** [review]
Hi @mahammad-mostafa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

power approve


## Highlights
Used react-router :+1: 
Added the delete button as requested by previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**